platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformSchedulersConfig.java

```diff
@@ -138,6 +138,9 @@ public record PlatformSchedulersConfig(
         @ConfigProperty(defaultValue = "SEQUENTIAL CAPACITY(60) UNHANDLED_TASK_METRIC")
                 TaskSchedulerConfiguration stateGarbageCollector,
         @ConfigProperty(defaultValue = "200ms") Duration stateGarbageCollectorHeartbeatPeriod,
+        @ConfigProperty(defaultValue = "SEQUENTIAL UNHANDLED_TASK_METRIC")
+                TaskSchedulerConfiguration signedStateSentinel,
+        @ConfigProperty(defaultValue = "1000ms") Duration signedStateSentinelHeartbeatPeriod,
```

Comment:
based on the `setMinimumPeriod` that was set in the old `SignedStateSentinal`, perhaps this should be 10 seconds? Unless there is a specific reason to speed up from the old rate

Reply:
cody mentioned 1 second.  I'll bump it to 10s to match the old configuration